tests/baselines/reference/docker/office-ui-fabric.log

```diff
@@ -80,6 +80,7 @@ Standard output:
 @uifabric/merge-styles:  PASS  src/Stylesheet.test.ts
 @uifabric/merge-styles:  PASS  src/extractStyleParts.test.ts
 @uifabric/merge-styles:  PASS  src/concatStyleSetsWithProps.test.ts
+@uifabric/merge-styles:  PASS  src/fontFace.test.ts
```

Comment:
_This test_ IDK why, but this is, like, only probabilisticly tested or something